docs/synthetics/http-monitor.md

```diff
@@ -42,6 +42,17 @@ If you disable this option, we **won’t save the response body** hence it **wil
 
 ![Save Response Body Disabled](../images/synthetics/response-body-disabled.png)
 
+## Bulk Adding HTTP Monitors
+
+When creating an HTTP Monitor, you can add multiple URLs on the same page. These URLs will inherit the same configuration settings applied during the creation process. To add URLs individually, click the '+ New Monitor' button for each one.
```

Comment:
you can add multiple URLs on the same page ==> you can add multiple URLs at once
These URLs will inherit the same configuration settings applied during the creation process ==> All configuration settings specified during the creation process will be applied to URLs added in bulk